### DISCUSSION TITLE
Why is the plain text version of a mailable not accessible?

### DISCUSSION THREAD

**Discussion by troygilbert** [laravel/framework#44281] [Q&A]
Mailables have a facility for building HTML and plaintext versions of themselves. Frustratingly, though, the methods are all protected. I can call render() to get the HTML version, but there doesn't seem to be any way to access the text version.

My use case is that we store the plain text version of the email as part of a log. Currently, I have to grab the HTML version and then do several very specific cleaning steps to strip HTML and cut out headers, etc.

**Comment by fredsal** [comment]
Feel free to make a PR for that
